support/1p.env

```diff
@@ -0,0 +1,2 @@
+PERIDIO_CERTIFICATE=op://avocado/qemuarm64-secureboot/certificate
+PERIDIO_PRIVATE_KEY=op://avocado/qemuarm64-secureboot/private-key
```

Comment:
I don't think we want to commit this file as it is specific to your personal vault structure (e.g. avocado).